tnb/validators.py

```diff
@@ -2,15 +2,15 @@
 
 
 class Validator(BaseClient):
-    def fetch_accounts(self):
+    def fetch_accounts(self) -> object:
```

Comment:
This method returns a `list`.

**Note**: The method documentation is also incorrect. 

```suggestion
    def fetch_accounts(self) -> list:
```